### PR TITLE
Convert yast-python-bindings to python3, also fix some examples

### PR DESCRIPTION
**Pull request by dmulder**

### CODE REVIEW

File: examples/Table-icons.py [hunk 1]
```diff
@@ -68,14 +75,23 @@ def main(self):
...
-      iconList = ycpbuiltins.sort(ycpbuiltins.filter(iconList) do |entry|
-        ycpbuiltins.regexpmatch(entry, "^.*.(png|jpg|PNG|JPG)$")
-      end)
-
-      copy.deepcopy(iconList)
+#      #TODO #FIXME add ycbbuiltins.filter()
+#      #TODO #FIXME add ycbbuiltins.sort()
+#      iconList = ycpbuiltins.sort(ycpbuiltins.filter(iconList) do |entry|
+#        ycpbuiltins.regexpmatch(entry, "^.*.(png|jpg|PNG|JPG)$")
+#      end)
```
Comment (jreidinger) [inline examples/Table-icons.py]: for me it looks like both TODO and FIXME is done, not? then we can remove this comments

File: examples/WindowID.py [hunk 2]
```diff
@@ -18,8 +18,8 @@ def main(self):
         caption,
         penguins,
         help,
-        Label.BackButton,
-        Label.NextButton
+        "Back", #Label.BackButton
```
Comment (jreidinger) [inline examples/WindowID.py]: this is a bit step back, as `Label.BackButton` is translated and should be promoted.

File: examples/Table-icons.py (hunk 1 above)
Comment (dmulder) [reply examples/Table-icons.py]: nope, these still aren't implemented (see src/ycpbuiltins.py)

File: examples/WindowID.py (hunk 2 above)
Comment (dmulder) [reply examples/WindowID.py]: It was missing the import for the Label module (fixing)

File: examples/Table-icons.py (hunk 1 above)
Comment (jreidinger) [reply examples/Table-icons.py]: ah, ok.

**Pull request merged**